src/lib/y2network/interfaces_collection.rb

```diff
@@ -58,5 +58,14 @@ def find(name)
     def add(name)
       interfaces.push(Interface.new(name))
     end
+
+    # Compares InterfacesCollections
+    #
+    # @return [Boolean] true when both collections are equal, false otherwise
+    def ==(other)
+      @interfaces - other.interfaces && other.interfaces == @interfaces
```

Comment:
Sorry, but I do not get the conditions.

Reply:
yep, probably something like `other.interfaces.sort == @interfaces.sort`